src/kakarot/accounts/eoa/library.cairo

```diff
@@ -43,6 +43,8 @@ namespace ExternallyOwnedAccount {
     }
 
     // @notice This function is used to initialize the externally owned account.
+    // @param kakarot_address The kakarot smart contract address. 
+    // @param evm_address The EVM address version of the kakarot address.
```

Comment:
I think that there is something unclear to you regarding the current kakarot architecture and consequently these parameters.
In the EVM, there are two types of accounts: Contract Account (smart contract) and Externally Owned Account (addresses controlled by wallets)
In kakarot, there is a smart contract deployed for each EVM address (see `IAccount` interface`). This contract, thus a Starknet contract, has a storage var `evm_address` to store the EVM address is contract is deployed for